bashbrew/bashbrew.sh

```diff
@@ -413,6 +414,7 @@ while [ "$#" -gt 0 ]; do
 						"$docker" rmi -f "$namespace/$repoTag-$dateStamp"
```

Comment:
Yeah, we don't create datestamped version without namespace so only need to remove $repoTag